src/widgets/arc/lv_arc.h

```diff
@@ -27,10 +27,14 @@ extern "C" {
 /**********************
  *      TYPEDEFS
  **********************/
+
+/**
+ * In which direction the indicator should grow.
+ */
 typedef enum {
-    LV_ARC_MODE_NORMAL,
-    LV_ARC_MODE_SYMMETRICAL,
-    LV_ARC_MODE_REVERSE
+    LV_ARC_MODE_NORMAL,      /**< Clock-wise */
+    LV_ARC_MODE_SYMMETRICAL, /**< Left/right from the midpoint */
```

Comment:
```suggestion
    LV_ARC_MODE_SYMMETRICAL, /**< Left/right from midpoint */
```
There are times, especially in code comments which we want to keep as brief as possible, when the word "the" is omitted, and it is usually only applied in cases of "labeling" and "documentation" where it needs to be brief.  It also applies in most cases in comments above functions (or function prototypes in `.h` files).  Note:  it does not apply to all "the"s and I don't remember the exact rule, but it comes naturally to me.  There are some "the"s that should be left in place.

As soon as I can, I will try to get you the excellent treatise on this by Bertrand Meyer.  He explains why it is grammatically correct to do this type of omission, without harming readability or clarity, but reducing the amount of effort required to understand the concept conveyed.